wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -190,18 +193,27 @@ def get_status(self) -> Status:
                     "TerminationByKubelet",
                 ]:
                     return Status("preempted")
+
+        now = time.time()
+        if self._is_container_creating(pod):
+            if now - self._last_msg_time > FAIL_MESSAGE_INTERVAL:
+                wandb.termlog(f"{LOG_PREFIX}Container is creating for job: {self.name}")
+                self._last_msg_time = now
+            return Status("starting")
         if pod.status.phase in ["Pending", "Unknown"]:
-            now = time.time()
             if self._fail_count == 0:
                 self._fail_first_msg_time = now
-                self._fail_last_msg_time = 0.0
             self._fail_count += 1
-            if now - self._fail_last_msg_time > FAIL_MESSAGE_INTERVAL:
+            if now - self._last_msg_time > FAIL_MESSAGE_INTERVAL:
+                minutes = round(
```

Comment:
This can sometimes lead to a negative wait time. (-1 minutes). We should prevent that

Reply:
I changed `round` to `int`, which gives 0 for numbers as low as -0.99.